### PR TITLE
JCSP-407: DC, Cloud: View issue types

### PR DESCRIPTION
**Pull request by rsaliiatlassian**

### CODE REVIEW

File: custom-vu/src/main/kotlin/jces1209/vu/page/admin/issuetypes/DcBrowseIssueTypesPage.kt [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        loadingPageExpectedCondition
+    )
+        .serverErrors()
+        .timeout(Duration.ofSeconds(30))
+        .build()
+
+    override fun waitForBeingLoaded(): DcBrowseIssueTypesPage {
+        falliblePage.waitForPageToLoad()
+        return this
+    }
```
Comment (NataliiaShavshyna) [inline custom-vu/src/main/kotlin/jces1209/vu/page/admin/issuetypes/DcBrowseIssueTypesPage.kt:29]: looks like the same as for cloud

**Review by NataliiaShavshyna — CHANGES_REQUESTED** [review]

File: custom-vu/src/main/kotlin/jces1209/vu/ScenarioSimilarities.kt [hunk 2]
```diff
@@ -153,6 +155,10 @@ class ScenarioSimilarities(
             issueNavigator = issueNavigator,
             switchViewsProbability = 0.75f
         ),
+        browseIssueTypes = BrowseIssueTypes(
+            meter = meter,
```
Comment (ysemenkov) [inline custom-vu/src/main/kotlin/jces1209/vu/ScenarioSimilarities.kt]: measure

File: custom-vu/src/main/kotlin/jces1209/vu/page/admin/issuetypes/BrowseIssueTypesPage.kt [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+) {
+    protected val loadingPageExpectedCondition: ExpectedCondition<Boolean> = ExpectedConditions.and(
+        ExpectedConditions.visibilityOfElementLocated(By.className("aui-page-panel")),
+        ExpectedConditions.visibilityOfElementLocated(By.id("add-issue-type")),
+        ExpectedConditions.visibilityOfElementLocated(By.id("issue-types-table")),
+        ExpectedConditions.visibilityOfElementLocated(By.xpath("//*[. = 'Bug']")),
+        ExpectedConditions.visibilityOfElementLocated(By.xpath("//*[. = 'Epic']")),
+        ExpectedConditions.visibilityOfElementLocated(By.xpath("//*[. = 'Story']")),
+        ExpectedConditions.visibilityOfElementLocated(By.xpath("//*[. = 'Task']")),
+        ExpectedConditions.visibilityOfElementLocated(By.xpath("//*[. = 'Sub-task']"))
```
Comment (ysemenkov) [inline custom-vu/src/main/kotlin/jces1209/vu/page/admin/issuetypes/BrowseIssueTypesPage.kt]: it is too general locators and is related to text
```suggestion
        presenceOfAllElementsLocatedBy(By.cssSelector("[data-issue-type-field]")),
        presenceOfAllElementsLocatedBy(By.cssSelector("[data-type]")),
        presenceOfAllElementsLocatedBy(By.className("operations-list")),
        visibilityOfElementLocated(By.className("search-entry"))
```

**Review by ysemenkov — CHANGES_REQUESTED** [review]

File: custom-vu/src/main/kotlin/jces1209/vu/page/admin/issuetypes/CloudBrowseIssueTypesPage.kt [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+class CloudBrowseIssueTypesPage(
+    private val jira: WebJira
+) : BrowseIssueTypesPage(jira) {
+
+    private val falliblePage = FalliblePage.Builder(
+        jira.driver,
+        and(
+            loadingPageExpectedCondition,
+            presenceOfElementLocated(By.xpath("//*[@data-testid = 'NavigationItem']")),
```
Comment (ysemenkov) [inline custom-vu/src/main/kotlin/jces1209/vu/page/admin/issuetypes/CloudBrowseIssueTypesPage.kt]:
```suggestion
            presenceOfAllElementsLocatedBy(By.xpath("//*[@data-testid = 'NavigationItem']")),
```

**Review by ysemenkov — CHANGES_REQUESTED** [review]

**Review by ysemenkov — APPROVED** [review]

**Review by NataliiaShavshyna — APPROVED** [review]

**Pull request merged**